app/views/components/view_application.scala.html

```diff
@@ -167,7 +178,7 @@ <h2>@messages("application.footer.sendingSamples.heading")</h2>
     <div class="print-document">
     <div class="grid-row print-header">
         <div class="column-one-third">
-            <img class="print-header__hmrc-logo" alt="HM Revenue and Customs" src="@createLink("images/hmrc_logo.jpg")" />
+            <img class="print-header__hmrc-logo" alt="HM Revenue and Customs" src="@createLink("images/hmrc_logo.jpg")"" />
```

Comment:
```suggestion
            <img class="print-header__hmrc-logo" alt="HM Revenue and Customs" src="@createLink("images/hmrc_logo.jpg")" />
```
I think this quote might be duplicated